lib/schedule/health/checkers/timepoints_checker.ex

```diff
@@ -18,6 +19,15 @@ defmodule Schedule.Health.Checkers.TimepointsChecker do
     timepoints_on_route_fn =
       Application.get_env(:skate_web, :timepoints_on_route_fn, &Schedule.timepoints_on_route/1)
 
-    timepoints_on_route_fn.(route_id) |> length() >= min_length
+    length = timepoints_on_route_fn.(route_id) |> length()
+    passfail = length >= min_length
+
+    if(!passfail) do
+      Logger.warning(
+        "Timepoints Checker failed on route #{route_id}. min_legth=#{min_length} length=#{length}"
```

Comment:
Here and a couple of other places: prefer `route_id=#{route_id}` instead of sentence-style formatting to make Splunk queries easier.